bin/collator/Cargo.toml

```diff
@@ -16,9 +16,11 @@ crate-type = ["cdylib", "rlib"]
 
 [dependencies]
 # thidr-party dependencies
-parity-util-mem = { version = "0.11.0", default-features = false, features = ["jemalloc-global"] }
+parity-util-mem = { version = "0.11.0", default-features = false, features = [
+  "jemalloc-global",
+] }
```

Comment:
Tbh, I'm not sure if I'm a fan of this _fixed_ format. It will take a few days to get used to :)

Reply:
It's the result of `taplo fmt`. but it seems frontier's Cargo.toml has one liner even though it uses taplo too. Please let me check

Reply:
Must be the `taplo.toml` in the project root.